src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/custom.py

```diff
@@ -1261,6 +1270,49 @@ def _get_log(url, user_name, password, log_file=None):
     r.release_conn()
 
 
+def _print_site_logs(api_string, number_of_lines, byte_limit, http, headers):
+    import sys
+    href_count = api_string.count('https')
+
+    std_encoding = sys.stdout.encoding
+    if href_count > 1:
+        logger.info('Will show last %i lines of logs per instance', number_of_lines)
+
+    for x in range(0, href_count):
+        index_of_path = api_string.index('\",\"path')
+        href_str = api_string[api_string.index("https"):index_of_path]
+        size = api_string[api_string.index('\"size\":') +
+                          len('\"size\":'):api_string.index(',\"href\"')]
+        logger.warning('\nSite Instance #%i', x + 1)
```

Comment:
If this is not super important, use `.info()`, this way, the default output will be pure app's own trace